File: power/technologies/python/fizzbuzz.py


class FizzBuzz:
    def output(self):
        for i in range(1, 101):
            string = ""
            if i % 3 == 0:
                string += "Fizz"
            if i % 5 == 0:
                string += "Buzz"
            print(i, string)


test = FizzBuzz()
test.output()

File: power/technologies/c#/Fizzbuzz.cs
using System;

class Driver
{
    static void Main(string[] args)
    {
        for (int i = 1; i <= 100; i++)
        {
            string phrase = "";
            if (i % 3 == 0)
            {
                phrase += "Fizz";
            }
            if (i % 5 == 0)
            {
                phrase += "Buzz";
            }
            Console.WriteLine(i + ":" + phrase);
        }

        Console.ReadLine();
    }
}